### PR TITLE
ref: Project structure

### PR DESCRIPTION
**Pull request by romtsn**
* Consolidates all apple targets under the `appleMain` sourceset
* Renames the package name (otherwise it clashes with the sentry-java sdk, whose package name is `io.sentry` as well)

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -44,6 +47,9 @@ kotlin {
         }
     }
     ios()
+    watchos()
+    tvos()
+    macosX64()
```
Comment (philipphofmann) [inline build.gradle.kts:52]: Oh yeah 💪

File: build.gradle.kts [hunk 2]
```diff
@@ -10,6 +10,7 @@ plugins {
 repositories {
     gradlePluginPortal()
     mavenCentral()
+    google()
```
Comment (philipphofmann) [inline build.gradle.kts:13]: `l`: Just curios. Why do we need the Google repo?

File: src/jsTest/kotlin/io/sentry/kotlin/multiplatform/SentryTestsJS.kt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package io.sentry.kotlin.multiplatform
+
+class SentryTestsJS {
+
+}
```
Comment (philipphofmann) [inline src/jsTest/kotlin/io/sentry/kotlin/multiplatform/SentryTestsJS.kt:5]: `l`:
```suggestion

}
```

**Review by philipphofmann — APPROVED** [review]
If it works, 🚢

File: src/jsTest/kotlin/io/sentry/kotlin/multiplatform/SentryTestsJS.kt (hunk 3 above)
Comment (romtsn) [reply src/jsTest/kotlin/io/sentry/kotlin/multiplatform/SentryTestsJS.kt:5]: ktlint needed deadly

File: build.gradle.kts (hunk 2 above)
Comment (romtsn) [reply build.gradle.kts:13]: cause `sentry-android` pulls in some androidx/android deps, and if we don't add it, gradle doesn't know where to pull them from

**Pull request merged**